docs/config/applicationProperties.rst

```diff
@@ -17,10 +17,13 @@ Let's have a brief look at it:
 
     [COMMON]
     useRoddyVersion=current                     # Use the most current version for tests
+    passEnvironment=false
+    baseEnvironmentScript=[ENVIRONMENT_FILE]
```

Comment:
This should also give an example. If required this usually will/can be `/etc/profile`.